build.gradle

```diff
@@ -71,6 +71,12 @@ subprojects { project ->
           if (it.requested.group == 'org.apache.xbean') {
             it.useVersion '4.3'
           }
+          if (it.requested.group == 'com.fasterxml.jackson.core'
+                  || it.requested.group == 'com.fasterxml.jackson.datatype'
+                  || it.requested.group == 'com.fasterxml.jackson.dataformat'
+                  || it.requested.group == 'com.fasterxml.jackson.module') {
+            it.useVersion '2.5.4'
```

Comment:
I'm almost certain that this is not the only module that is going to be all over the place in versions in the graph. If the uberjar builder can't successfully deal with that then it is going to need an alternative solution than just explicitly version managing dependencies because that will get way out of hand